### ISSUE TITLE
"Async iterators and generators" not findable on website?

### ISSUE THREAD

**Issue by rektide** [Fyrd/caniuse#4948]
Hello. I added #4105 quite a while ago, hopefully resolving #3690.

However I can't seem to find it on caniuse.com . Searching for "Async" doesn't show me the feature in the search results list. Searching for the full description "Async iterators and generators" doesn't show it.

It seems like something is wrong but I don't know what it is. I'd love to be able to see this feature on caniuse.com. Any ideas what the issue is?

**Comment by Fyrd** [comment]
Yes, the data is still pending review before it's appears to the site to ensure correctness. Apologies that it's been a while. If it's possible for you to provide a test case to check browser support that would help speed things up.
